src/main/java/sparksoniq/spark/DataFrameUtils.java

```diff
@@ -93,24 +93,47 @@ public static void registerKryoClassesKryo(Kryo kryo)
      * @param trailingComma          boolean field to have a trailing comma
      * @return comma separated variables to be used in spark SQL
      */
-    public static String getSQL(
+    public static List<String> getColumnNames(
```

Comment:
We are using this a lot with: DataFrameUtils.getColumnNames(inputSchema, -1, null)

Maybe it's worth overloading the method :)